asynclibrary.resource

```diff
@@ -0,0 +1,13 @@
+*** Settings ***
+Library  AsyncLibrary/AsyncLibrary.py  WITH NAME  ASYNCLIBRARY
```

Comment:
First of all, thanks that you are trying to help, but I have a couple of issues with your proposal.

When I understand it correctly, you stumble only across all the problems, because you did not install the library as a python module as should be the case. So in other words, was the code installed via one of the methods proposed. Also did you try the examples?

Also the resource file here, I do not see an added value when you follow the example and the unit test in the test folder. Robot Framework 6.0.1 is also tested in the CI/CD pipeline. Thus this is working.

Further, pylint will complain about the changes you introduced as the import is not correctly done as part of your changes.

Regarding the introduction of AsyncLibrary.py file. Was the code actually changed and what or was the code only moved?

Reply:
Ok, RF 6.1.1 was not part of the CI/CD pipeline test, yet. Just added that tests are also executed against it and everything is working as expected. No changes required. You can look at the last actions what is all tested or easier in the pipeline file. Here an example of ubuntu latest, python 3.8 and RF 6.1.1
https://github.com/rlehfeld/robotframework-async-keyword/actions/runs/11157689805/job/31012453021